src/main/plugins/org.dita.base/xsl/preprocess/conrefImpl.xsl

```diff
@@ -487,55 +494,54 @@ See the accompanying LICENSE file for applicable license.
     <xsl:param name="current-relative-path" tunnel="yes" as="xs:string"/>
     <xsl:param name="conref-filename" tunnel="yes" as="xs:string?"/>
     <xsl:param name="conref-ids" tunnel="yes" as="xs:string*"/>
-    <xsl:attribute name="href">
```

Comment:
Use `xsl:copy` to output the attribute with the same value instead of generating all cases inside a common `xsl:attribute`